assemble/bin/accumulo-cluster

```diff
@@ -439,7 +448,7 @@ function stop_tservers() {
 
   echo "Cleaning tablet server entries from zookeeper"
   if ! isDebug; then
-    ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -tservers
+    ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -tservers -group "$groups"
```

Comment:
In the case where a group is not specified, then `$groups` is going to be all groups. Does ZooZap handle that correctly?

Reply:
Good catch, moved that section up to the last for loop so it's handled correctly.